presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -110,10 +110,13 @@ std::shared_ptr<connector::ColumnHandle> toColumnHandle(
     const protocol::ColumnHandle* column) {
   if (auto hiveColumn =
           dynamic_cast<const protocol::HiveColumnHandle*>(column)) {
+    // TODO(spershin): Should we pass something different than 'typeSignature'
+    // to 'hiveType' argument of the 'HiveColumnHandle' constructor?
     return std::make_shared<connector::hive::HiveColumnHandle>(
         hiveColumn->name,
         toHiveColumnType(hiveColumn->columnType),
         stringToType(hiveColumn->typeSignature),
+        stringToType(hiveColumn->typeSignature),
```

Comment:
I believe this should come from `HiveType hiveType = {};`

Reply:
@mbasmanova 
I thought so too, but it fails 14 queries in TestPrestoNativeGeneralQueriesJSON.
So I resorted to the default (what the backwards compatibility does right now).
Later we can put there what should be there.

Reply:
I will take care. Thanks!